apps/api/src/domain/models/vocabulary-list/commands/translate-vocabulary-list-name/translate-vocabulary-list-name.command.ts

```diff
@@ -0,0 +1,30 @@
+import { ICommandBase, LanguageCode } from '@coscrad/api-interfaces';
+import { Command } from '@coscrad/commands';
+import { NestedDataType, NonEmptyString } from '@coscrad/data-types';
+import { LanguageCodeEnum } from '../../../../../domain/common/entities/multilingual-text';
+import { VocabularyListCompositeId } from '../create-vocabulary-list';
+
+@Command({
+    type: 'TRANSLATE_VOCABULARY_LIST_NAME',
```

Comment:
Can we try exporting this as a constant as we've been doing?

Reply:
done